util/timeutil/isoduration.go

```diff
@@ -164,11 +164,15 @@ func ParseISODuration(s string) (d ISODuration, err error) {
 
 func (dur ISODuration) MarshalGQL(w io.Writer) {
 	if dur == (ISODuration{}) {
-		io.WriteString(w, "null")
+		if _, err := io.WriteString(w, "null"); err != nil {
+			panic(err)
+		}
 		return
 	}
 
-	io.WriteString(w, `"`+dur.String()+`"`)
+	if _, err := io.WriteString(w, `"`+dur.String()+`"`); err != nil {
+		panic(err)
+	}
```

Comment:
^^

Reply:
done.  set to ignore the error.  I only set to panic based on looking at the surrounding functions, but i've updated it to ignore the error.